components/kyma-environment-broker/internal/broker/instance_create.go

```diff
@@ -74,12 +75,20 @@ func (b *ProvisionEndpoint) Provision(ctx context.Context, instanceID string, de
 		return domain.ProvisionedServiceSpec{}, apiresponses.NewFailureResponse(err, http.StatusBadRequest, errMsg)
 	}
 
+	region, found := middleware.RegionFromContext(ctx)
+	if !found {
+		err := errors.New("No region specified in request.")
+		return domain.ProvisionedServiceSpec{}, apiresponses.NewFailureResponse(err, http.StatusInternalServerError, "provisioning")
+	}
+
 	provisioningParameters := internal.ProvisioningParameters{
 		PlanID:     details.PlanID,
 		ServiceID:  details.ServiceID,
 		ErsContext: ersContext,
 		Parameters: parameters,
+		Region:     region,
 	}
+
 	logger.Infof("Starting provisioning runtime: Name=%s, GlobalAccountID=%s, SubAccountID=%s", parameters.Name, ersContext.GlobalAccountID, ersContext.SubAccountID)
```

Comment:
please, add a region to log (or there is a place where you are logging region but I didn'f find it)

Reply:
done :)